frontend/src/components/Experiment/Experiment.test.jsx

```diff
@@ -23,7 +23,7 @@ vi.mock('../../util/stores', () => ({
     __esModule: true,
     default: (fn) => {
         const state = {
-            session: 1,
+            session: { id: 1 },
             participant: 'participant-id',
```

Comment:
It would be nice to be consistent: either just pass the id as a string / integer, or pass an object for both session and participant.

Reply:
They stores already carry objects with ids so it only makes sense that the mock stores do the same.